### PR TITLE
Console output has been improved for `Additional info` section

### PR DESCRIPTION
**Pull request by andrewst**

### CODE REVIEW

**Comment by andrewst** [comment]
@UnikZ please review

File: analysis_options.yaml [hunk 1]
```diff
@@ -46,7 +46,7 @@ linter:
     - prefer_is_not_empty
     # - public_member_api_docs
     #- slash_for_doc_comments
-    - sort_constructors_first
+    #- sort_constructors_first
```
Comment (UnikZ) [inline analysis_options.yaml]: Maybe, we have to remove comments instead of add a new one?

File: lib/configure_logging_for_browser.dart [hunk 2]
```diff
@@ -48,7 +48,10 @@ class ConfigureLoggingForBrowser {
   }
 
   static void listenJsErrors(LoggingService loggingService,
-    {bool preventDefault: true, html.Window window, Protector infiniteLoopProtector, bool handleGlobalJsErrors: false}) {
+      {bool preventDefault: true,
+      html.Window window,
+      Protector infiniteLoopProtector,
+      bool handleGlobalJsErrors: false}) {
```
Comment (UnikZ) [inline lib/configure_logging_for_browser.dart]: You can add a comma before `}` symbol and use dartfmt to make vertical alignment

File: pubspec.yaml [hunk 3]
```diff
@@ -14,11 +14,10 @@ dependencies:
...
 dev_dependencies:
-  test: '>=0.12.15 <2.0.0'
-  coverage: '>=0.7.9 <2.0.0'
-  mockito: '>=3.0.0-beta+1 <4.0.0'
-  quiver: '>=0.21.4 <2.0.0'
-  dart_dev: '>=1.7.2'
-  build_runner: '>=0.9.1 <2.0.0'
-  build_test: '>=0.10.2 <2.0.0'
-  build_web_compilers: '>=0.4.0 <2.0.0'
+  test: ^1.3.3
```
Comment (UnikZ) [inline pubspec.yaml:17]: Oh, nice! 👍

**Comment by UnikZ** [comment]
@andrewst, I've left some comments

**Pull request merged**